### ISSUE TITLE
ZED Core dumped

### ISSUE THREAD

**Issue by eduardohenriquearnold** [stereolabs/zed-opencv#46]
I have installed ZED SDK v1.1.1 (for my app code compability) on a Ubuntu 16.04 Docker container having CUDA 8.0. All dependency are handled but when I try to open ZED Explorer or ZED Depth Viewer I simply get 

> QXcbConnection: Could not connect to display 
> Aborted (core dumped)

The ZED diagnostic log is as follows:

>  *******************************
> *** OS Version ***
> Distributor ID:	Ubuntu
> Description:	Ubuntu 16.04.2 LTS
> Release:	16.04
> Codename:	xenial
> 
> Linux 5f17203f722d 4.10.0-30-generic #34~16.04.1-Ubuntu SMP Wed Aug 2 02:13:56 UTC 2017 x86_64 x86_64 x86_64 GNU/Linux
> 
> 
> *******************************
> *** Hardware ***
> 
> 
> *******************************
> *** USB Connection List ***
>  Detected USB3.0 component : 0
>  Detected ZED with PID/VID : 0
>  USB : 
> 
> *******************************
> 
> *******************************
> *** ZED Capabilities***
> ERROR /dev/videoX : didn't find suitable videodevice -1
>  ZED Serial Number : -1
>  ZED Firmware Number : -1
> 
> *******************************
> *** CUDA Capabilities***
> 
>  *** CUDA Check *** 
> (SMI) : Driver Version                      : 384.59
> (SMI) :     Driver Model
> (SMI) :     Performance State               : P8
> cuDeviceGetCount : OK : found 1 NVIDIA Graphic card. Taking dev 0
> cuDeviceGetName : OK : GeForce GTX 1060 6GB
> cuDeviceComputeCapability :  Compute Cap :  6 1 -- must be between 2.0 and 5.3 
> cuDriverGetVersion :  CUDA Driver version : 9000 . Must be >=7500 for CUDA 7.5 toolkit.
> cuDeviceTotalMem:   CUDA Memory  4096 MBytes (4294967295 bytes)

I suppose the core dumped error may have been caused by the compute capability not being between 2.0 and 5.3 but https://github.com/stereolabs/zed-ros-wrapper/issues/84 suggests that it should work with CUDA 8.0 (which happens to be the version I am using). 

I would also like to note that I do not have the camera connected, I just wish to playback some SVO files.

**Comment by eduardohenriquearnold** [comment]
I have managed to avoid the core dump error by changing my ZED initalization code, using pointers instead of object instantiation, ie ``zed = new sl::zed::Camera(argv[2]);``
However another problem arised. I am trying to open SVO files recorded with SDK version 0.9.4. When I try to open them I get the following error:

``` 
ERROR : Since the version 1.0.0 of the SDK the VGA resolution is 672*376, it seems you try to open an old SVO or you do not have update your firmware.
Unable to init the ZED:ZED_WRONG_FIRMWARE
```
Since I cannot use SDK version 0.9.4 because CUDA 7.5 won't work with my GTX1060 GPU, how should I proceed to make this work? I do not have an older GPU I can use, so I need to make it work with this one.
